cmd/prometheus/main.go

```diff
@@ -141,7 +141,6 @@ var (
 
 func init() {
 	// This can be removed when the default validation scheme in common is updated.
-	model.NameValidationScheme = model.UTF8Validation
```

Comment:
Should we keep it just for safety? No harm, but will give consistent behaviour if we accidently build Prometheus with old common or common global gets reverted.

(just for prom and promtool main.go)